ide/hudson/src/org/netbeans/modules/hudson/impl/HudsonJobBuildImpl.java

```diff
@@ -111,7 +111,7 @@ public Collection<? extends HudsonMavenModuleBuild> getMavenModules() {
         return modules;
     }
 
-    @Messages({"# {0} - job/module display name", "# {1} - build number", "HudsonJobBuildImpl.display_name={0} #{1,number,#}"})
+    @Messages({"# {0} - job/module display name", "# {1} - build number", "JenkinsJobBuildImpl.display_name={0} #{1,number,#}"})
```

Comment:
We should not change this because the class name is used. 
Maybe, there should be an error. (see: the return statement)